src/components/ViewRequest/Methods/LinkedInJobPost.tsx

```diff
@@ -47,15 +43,49 @@ const ViewRequestLIJobPostDetails = (props: { data: RPARequest }) => {
         {props.data.linkedinQualifications?.map((qual) => qual + "\n")}
       </Text>
 
-      <Label weight="semibold" htmlFor="dcwf">
-        Certifications/Licensure
-      </Label>
-      <Text style={{ whiteSpace: "pre-wrap" }} id="dcwf">
-        {props.data.dcwf?.map((cert) => {
-          const found = DCWFCodes.find((element) => element.Code === cert);
-          return cert + " " + found?.Role + "\n";
-        })}
-      </Text>
+      <fieldset style={{ gridColumn: "1 / 3" }}>
+        <legend>Defense Cyber Work Force (DCWF)</legend>
+        <table>
+          <thead>
+            <tr>
+              <th>Role</th>
+              <th>Proficiency</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                {props.data.dcwf +
```

Comment:
These return and display as "undefined" in the request view screen if they haven't selected a code.